composer.json

```diff
@@ -14,8 +14,8 @@
         "twig/extensions": "~1.0",
         "symfony/swiftmailer-bundle": "~2.3",
         "symfony/monolog-bundle": "~2.4",
-        "sensio/distribution-bundle": "^3.0.12",
-        "sensio/framework-extra-bundle": "^3.0.2",
+        "sensio/distribution-bundle": "^3.0",
+        "sensio/framework-extra-bundle": "^3.0",
```

Comment:
Because of current composer that requires distribution-bundle afaik > 3.0.18 (event classes changes). I had post-install script errors w/o this change.

Do you see any problem with locking to 3.0.x? It seems to be working fine for me in preliminary tests.